google/cloud/bigquery/magics/magics.py

```diff
@@ -239,6 +242,30 @@ def project(self):
     def project(self, value):
         self._project = value
 
+    @property
+    def client_options(self):
```

Comment:
We might actually want two client options, one for the BigQuery client and one for the BigQuery Storage client.